packages/web-components/src/components/salable-pricing-table/salable-pricing-table.tsx

```diff
@@ -60,6 +60,12 @@ export class SalablePricingTable {
    * The ID of the member who will manage the license.
    **/
   @Prop() member!: string;
+  /**
+   * The owner of the subscription
```

Comment:
```suggestion
   * The owner of the subscription.
```

Super minor but can we add the `.` so it doesn't get joined together into what looks like one sentence in the READMEs.

<img width="219" alt="Screenshot 2025-02-20 at 16 05 59" src="https://github.com/user-attachments/assets/04d0aca2-0db3-4c47-a5df-2682d54ce232" />